includes/multi-currency/MultiCurrency.php

```diff
@@ -1441,28 +1441,56 @@ public function is_multi_currency_settings_page(): bool {
 		);
 	}
 
+	/**
+	 * Function used to compute the customer used currencies, used as internal callable for get_all_customer_currencies function.
+	 *
+	 * @return array
+	 */
+	public function callable_get_customer_currencies() {
+		global $wpdb;
+
+		$currencies  = $this->get_available_currencies();
+		$query_union = [];
+
+		if ( class_exists( 'Automattic\WooCommerce\Utilities\OrderUtil' ) &&
+					\Automattic\WooCommerce\Utilities\OrderUtil::custom_orders_table_usage_is_enabled() ) {
+			foreach ( $currencies as $currency ) {
+				$query_union[] = sprintf(
+					'SELECT "%1$s" AS currency_code, EXISTS(SELECT currency FROM %2$s WHERE currency="%1$s" LIMIT 1) AS exists_in_orders',
+					$currency->code,
+					$wpdb->prefix . 'wc_orders'
+				);
+			}
+		} else {
+			foreach ( $currencies as $currency ) {
+				$query_union[] = sprintf(
```

Comment:
Maybe using `wpdb:prepare` is better in this case?
https://developer.wordpress.org/reference/classes/wpdb/prepare/